resources/instance_systemd.rb

```diff
@@ -93,9 +93,6 @@
   def create_init
     include_recipe 'memcached::_package' unless new_resource.binary_path
```

Comment:
I'm really not a fan of including recipes in resources. It's allowed though.

Reply:
Same but trying to keep compatibility. Might go for a major version bump and clean it up further